Small change: Improve clarity in docs/06-responses.md (#2853)
rwf2/Rocket: docs/guide/06-responses.md

struct WrappingResponder<R>(R);
```

> A wrapping responder modifies the response returned by `R` before sending the modified response. For instance, Rocket provides `Responder`s in the

Reviewer: ```suggestion
A wrapping responder takes a response from `R`, modifies it, and returns it. For instance, Rocket provides `Responder`s in the
```
